File: fizzbuzz/fizzbuzz.py

def fizzbuzz(max):
    for val in range(1, max + 1):
        if (val % 15 == 0):
            print 'fizzbuzz'
        elif (val % 5 == 0):
            print 'buzz'
        elif (val % 3 == 0):
            print 'fizz'
        else:
            print val

fizzbuzz(100)

File: fizzbuzz/fizzbuzz.php
<?php

function fizzbuzz($max) {
  foreach (range(1, $max) as $value) {
    if ($value % 15 === 0) echo 'fizzbuzz';
    elseif ($value % 3 === 0) echo 'fizz';
    elseif ($value % 5 === 0) echo 'buzz';
    else echo $value;
    echo PHP_EOL;
  }
};

fizzbuzz(100);
